src/dotnet/HQ.Server/Services/HolidayServiceV1.cs

```diff
@@ -19,9 +20,15 @@ public class HolidayServiceV1
 {
     private readonly HQDbContext _context;
 
-    public HolidayServiceV1(HQDbContext context)
+    private readonly ILogger<ProjectStatusReportServiceV1> _logger;
+    private const string VacationChargeCode = "S1001"; // S1001 is the code for vacation
```

Comment:
This is the wrong charge code, Holiday is different from Vacation. We should also do a lookup based on the name of the project associated with the charge code to keep it generic - Check where project name like "holiday". (Look at how dashboard is calculating vacation hours and look it up the same way for Holiday.